vendor/engines/saml_authentication/lib/saml_authentication/devise_configurator.rb

```diff
@@ -30,6 +30,18 @@ def configure!
 
     private
 
+    def saml_updater
+      Settings.saml.user_updater_class_name.presence.try(:constantize).try(:new) || SamlAuthentication::UserUpdater.new
+    end
+
+    def saml_create_user?
+      if !Settings.saml.create_user.nil?
```

Comment:
I would probably flip the conditional. It gets rid of a double negative, and I think it clarifies that it's a default.

```ruby
if Settings.saml.create_user.nil?
  true
else
  Settings.saml.create_user
end
```

or an alternative construction:

```ruby
Settings.saml.create_user = true if Settings.saml.create_user.nil?
Settings.saml.create_user
```